.github/workflows/backend-tests-on-docker.yml

```diff
@@ -107,6 +107,7 @@ jobs:
           --ignore tests/integration/workflows/test_workflows_cli.py \
           --ignore tests/integration/workflows/test_workflow_utils.py \
           --ignore tests/integration/workflows/test_workflow_core_selection.py \
+          --ignore tests/integration/workflows/test_workflow_non_core_selection.py \
```

Comment:
why ignoring it?

Reply:
because we split tests into 2 workflows, so no sense in running it in both